test/test_audio.rb

```diff
@@ -10,8 +10,8 @@ class TestAudio < Minitest::Test
   def test_sample
     skip_on_appveyor
 
-    sound = Gosu::Sample.new(media_path("0614.ogg"))
-    channel = sound.play(1, 1, true)
+    sound = Gosu::Sound.new(media_path("0614.ogg"))
+    channel = sound.play(volume: 1, speed: 1, looping: true)
```

Comment:
I thought, that if we already "drastically" change the naming part we might as well make this a hashed list as well?

Reply:
I know that Ruby 2.1+ supports some form of named arguments on the language level, and I've been meaning to look into this. I think your proposed interface should be the long-term goal, but I would still provide the old interface for compatibility (so that we can just `Sample = Sound` for now; also makes the C++ bridging a bit easier).

Reply:
Also, this would be a good way to get rid of `play_pan` altogether. I think the issue is that `play` used to be louder than `play_pan(0)` at some point in time; I don't think that's true anymore, and even if it is, then the default value for `pan` could just be `nil` instead of `0` to distinguish both cases.

Reply:
I thought about doing it the same way you handled Image.from_text. There you already deprecated a argument list and hinted towards the option-hash. if we internally use a Hash (to be backward compat to Ruby <2.0) or named parameters is just a matter of personal preference (and maybe a bit about more or less code). Ruby 2.1 only introduced _require-named-parameters_ which also reduces a bit of code by doing some checks and raising an error automatically.
So I guess you're the one who defines whats the lowest Ruby Version Gosu should support?

Reply:
I was planning to raise the minimum version to 2.3 right after Gosu 1.0. Some Gosu-related gems are still only compatible with Ruby 1.9.3(!) as far as I know. But I don't think we really need *required* arguments in `Sound#play` so a simple hash sounds fine as in `Image.from_text` sounds good enough :)

Reply:
Another random thought. We should test whether `play_pan(0)` and `play` sound the same *with a stereo sample*, not just good old Beep.wav :)